gloo/common/CMakeLists.txt

```diff
@@ -1,18 +1,25 @@
 set(GLOO_COMMON_SRCS
-  "${CMAKE_CURRENT_SOURCE_DIR}/linux.cc"
   "${CMAKE_CURRENT_SOURCE_DIR}/logging.cc"
   )
 
 set(GLOO_COMMON_HDRS
   "${CMAKE_CURRENT_SOURCE_DIR}/aligned_allocator.h"
```

Comment:
(Also mentioned in #197), does `aligned_allocator.h` need to be moved as well, as `posix_memalign` is a linux specific call?

Reply:
macOS does have `posix_memalign`.